java/src/main/java/ai/onnxruntime/OrtProvider.java

```diff
@@ -11,6 +11,7 @@
 public enum OrtProvider {
   CPU("CPUExecutionProvider"),
   CUDA("CUDAExecutionProvider"),
+  ROCM("ROCMExecutionProvider"),
```

Comment:
Remove this line since it is already defined in line#26.

Reply:
removed